News/App.xaml.cs

```diff
@@ -100,11 +115,54 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
         /// </summary>
         /// <param name="sender">Источник запроса приостановки.</param>
         /// <param name="e">Сведения о запросе приостановки.</param>
-        private void OnSuspending(object sender, SuspendingEventArgs e)
+        private  void OnSuspending(object sender, SuspendingEventArgs e)
         {
+            WriteUserCheckedListElements();//json user data to file
```

Comment:
Make this call awaited

Reply:
OnSuspending can't be overrided with async